conda/environment-test-3.10.yml

```diff
@@ -1,14 +1,22 @@
-name: cunumeric-test
+name: legate-core-test
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python=3.10
 
   # build
-  - cutensor>=1.3.3
+  - git
   - nccl
+  - make
+  - zlib
+  - cmake>=3.24
+  - ninja
```

Comment:
Do we need both make and ninja?

Reply:
Some people may prefer make over ninja, but I'm fine going w/ only ninja.